config/vsc_hortense.py

```diff
@@ -64,6 +63,9 @@ def command(self, job):
                     'scheduler': 'slurm',
                     'prepare_cmds': [prepare_eessi_init, common_eessi_init()],
                     'access': hortense_access + ['--partition=cpu_rome'],
+                    'sched_options': {
+                        'shed_access_in_submit': True,
```

Comment:
I think this is a typo: should be `sched_access_in_submit` https://reframe-hpc.readthedocs.io/en/stable/config_reference.html#config.systems.partitions.sched_options.sched_access_in_submit